docs/configmaps.md

```diff
@@ -35,7 +35,7 @@ data:
     env: []
 ```
 
-Please note, the `labels` must match the provided `$CONFIGMAP_LABELS`, so the injector can discover these ConfigMaps. Additionally, make sure the `namespace` jives with the `$CONFIGMAP_NAMESPACE` (or if omitted, the ConfigMap is in the same namespace as the sidecar injector).
```

Comment:
'Jive' is not defined as "in accord with," but has been used as such since the 1940s. I think it makes sense, just might not be the most correct English :)

Reply:
This bit of vernacular seems to be regional dialect that should technically be `jibe` (or any other phrase meaning "to agree with"). Chalk this up to [betacism](https://en.wikipedia.org/wiki/Betacism) in midwestern english, making `jive with` and `jibe with` interchangable.

This change ought to read more like the following, to be less idiomatic:
> Additionally, make sure the `namespace` agrees with the `$CONFIGMAP_NAMESPACE`...